arangod/Statistics/StatisticsFeature.cpp

```diff
@@ -337,14 +350,22 @@ void StatisticsFeature::collectOptions(std::shared_ptr<ProgramOptions> options)
 
   options->addOption("--server.statistics",
                      "turn statistics gathering on or off",
-                     new BooleanParameter(&_statistics),
-                     arangodb::options::makeDefaultFlags(arangodb::options::Flags::Hidden));
+                     new BooleanParameter(&_statistics));
+
   options->addOption("--server.statistics-history",
                      "turn storing statistics in database on or off",
-                     new BooleanParameter(&_statisticsHistory),
-                     arangodb::options::makeDefaultFlags(arangodb::options::Flags::Hidden))
```

Comment:
Just curious: why do we remove arangodb::options::makeDefaultFlags(arangodb::options::Flags::Hidden)) here?

Reply:
I think it is a useful option that deserves more attention. Using it can greatly reduce idle write load on an instance, so making people aware of that it exists is useful IMHO.